tx_test.go

```diff
@@ -120,6 +123,74 @@ func TestNewTxFromBytes(t *testing.T) {
 	})
 }
 
+// This test reads a sample block from a file, but normally
+// we would get the block directly from the node via a REST GET...
+/*
+	resp, err := http.Get(fmt.Sprintf("%s/rest/block/%s.bin", b.client.serverAddr, blockHash))
+	if err != nil {
+		return nil, fmt.Errorf("Could not GET block: %v", err)
+	}
+
+	if resp.StatusCode != 200 {
+		defer resp.Body.Close()
+
+		data, err := ioutil.ReadAll(resp.Body)
+
+		if err != nil {
+			return nil, fmt.Errorf("failed to read response body: %w", err)
+		}
+
+		return nil, fmt.Errorf("ERROR: code %d: %s", resp.StatusCode, data)
+	}
+
+	Then use resp.Body as in the test below
+
+*/
+func TestNewTxFromReader(t *testing.T) {
+	var err error
+
+	f, err := os.Open("./block.bin")
```

Comment:
Could this file be removed and instead a reader be built from a `[]byte` just to keep the root filesystem clean?

Reply:
Actually, moving it to `testing/data` would be enough since the file is pretty big